### DISCUSSION TITLE
Welcome to idlecombos Discussions!

### DISCUSSION THREAD

**Discussion by djravine** [djravine/idlecombos#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by raxele123** [comment]
IdleCombos Suggestions:

1. Dark Mode please my eyes hurt.

2. "Use Blacksmith Contracts" please save the amount of contracts you use. By save I mean when you insert a certain champion's ID, the next time you use a blacksmith contract it defaults to that champion. I would like the same to be applied to the amount of contracts.

3. Hotkeys that send me straight to the "How many Blacksmith Contracts?" For every type of blacksmith contract. Also the choice to change key bindings please.

4. "Buy Event" and "Open Event" chests that is. Insert the event chest id and buy/open event chests though IdleCombos rather than SoulReaver. Though SoulReaver is just buy not open. 

5. Increase the font size of the "Champ IDs" please. Or maybe it's just me.

Please and thank you.

**Reply by djravine** [reply]
Moved to new discussion, rather than a comment. https://github.com/djravine/idlecombos/discussions/13

**Comment by djravine** [comment]
## For Future Reference
Please add each new suggestion as separate discussions.
This will allow me to create an issue based off the discussion.
Also, this allows the community to upvote specific requests.
